airflow/contrib/sensors/gcs_sensor.py

```diff
@@ -15,7 +15,6 @@
 from airflow.operators.sensors import BaseSensorOperator
 from airflow.utils.decorators import apply_defaults
 
-
```

Comment:
Why removed? Need to keep 2 empty lines before class declaration (pep8).

Reply:
i can change it but is this the only change required ? If there is anything else I would rather know now so I can change it once as this PR has been open for quite a while